### DISCUSSION TITLE
Technic Hub connected but unable use

### DISCUSSION THREAD

**Discussion by M0ochi** [pybricks/support#1652] [Q&A]
In dire need of help from any kind souls. I've downloaded the pybricks firmware onto my technic hub and managed to connect it to my PC via bluetooth. Issue that I'm facing now is that I am unable to get my motors to run despite trying various combinations of setups, inputs and outputs. Is there something I'm doing wrong? I was able to run the motors using the Powered Up App on android. 

Any help is greatly appreciated thank you!

P.S: I'm using the free trail of the block coding version

**Comment by laurensvalk** [comment]
Thanks for trying out the blocks.

Could you post a screenshot of the program that isn't working?

**Reply by laurensvalk** [reply]
How about something like the following? You can download this ready-made program <a href="https://pybricks.com/assets/programs/hello_world.py" download>here</a>.

You can import it like this:

![image](https://github.com/pybricks/support/assets/12326241/6af3abe6-b720-4a0f-9fe2-86eb64754425)

Example:

![image](https://github.com/pybricks/support/assets/12326241/643e387a-fa70-4321-85ea-e04e74529627)

**Reply by M0ochi** [reply]
thanks for the quick response! i tried downloading the program you shared but still got the same error message that i'm my motor isn't connected. tried downloading the allow_missing_motors file too but am unable to paste over the codes since i'm using the block coding version.

 ![image](https://github.com/pybricks/support/assets/171169351/76637a45-9833-420a-9adf-b7a2d43dec52)

**Reply by laurensvalk** [reply]
Did you plug in a motor into Port D? Can you share a photo of what you built?

There's no need to add any extra python files. Those just happened to be in my file browser -- sorry if that made it more confusing.

**Reply by laurensvalk** [reply]
Is it one of these?

![image](https://github.com/pybricks/support/assets/12326241/81569712-39e9-439e-a251-7512037669c5)

Or these? (Note the different kind of setup block)

![image](https://github.com/pybricks/support/assets/12326241/fe1554f3-447b-4f85-9e0c-46dabbf878a8)

**Reply by M0ochi** [reply]
yes i tried plugging in 4 motors to all the ports without any axles or builds connected to them that might prevent the motor from running but to no avail. i am using motors without rotation sensors with the correct "block" chosen with every variation possible in my coding but am still unable to get the motor to function.

**Reply by laurensvalk** [reply]
The motor without rotation sensors can't measure rotation, so running by a given amount of degrees is not possible with those.

You might want to do something like this:

![image](https://github.com/pybricks/support/assets/12326241/7c07f89d-bff6-466c-afbd-7e21dc30fc9d)

What programs have you tried with the DC Motor that didn't work?
